PopcornFX/README.md

```diff
@@ -0,0 +1,207 @@
+# PopcornFX Assets - O3DE Multiplayer Sample
+
+The Multiplayer Sample includes nine visual FX assets created with PopcornFX. This README documents the these FX assets so that you can customize them or deconstruct them to learn how to build PopcornFX assets of your own.
+
+The PopcornFX project containing all the source assets can be found in the following directory:
+`<multiplayer-sample-root>/PopcornFX/PopcornProject.pkproj`.
+
+## Requirements
+
+The **PopcornFX** Gem depends on the following additional Gems:
+
+* **Atom**
+* **Script Canvas**
+* **EMotionFX**
+* **PhysX**
+* **LyShine**
+* **Starting Point Input**
+
+## General information
+
+Some of the FX are split into multiple parts (for example, Blast, Projectile, and Explosion). Splitting the FX makes it easier to integrate the FX into varied gameplay scenarios.
+
+Many of the FX have a **Color Intensity** property that you can modify to adjust the emissive effect. The emissive effect requires that the **PostFX Layer** and **Bloom** components are added to the scene.
+
+<img src="readme_media/image1.png">
```

Comment:
Images don't work in preview. Should we be using markdown versions ie ![alt text](image path)?

Reply:
The problem is LFS. I'm using img tags because I need to size some images inside of tables. I've modified the `.gitattributes`, but need to see what I can do about drastically reducing media sizes.

Reply:
This should all be fixed. I have to use GIFs, but I have crunched them down to the smallest possible size while maintaining decent quality. Because I resized all the images I was able to switch to standard markdown tags for images.